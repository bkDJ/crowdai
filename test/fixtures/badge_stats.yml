# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html

one:
  badge_id: 1
  badge_count: 1
  participant_count: 1
  percentile: 1.5

two:
  badge_id: 1
  badge_count: 1
  participant_count: 1
  percentile: 1.5
