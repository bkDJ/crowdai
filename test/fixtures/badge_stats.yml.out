one:
  badge_id: 1
  badge_count: 1
  participant_count: 1
  percentile: 1.5

two:
  badge_id: 1
  badge_count: 1
  participant_count: 1
  percentile: 1.5
